dbus-opendtu.py

```diff
@@ -336,50 +346,86 @@ def _get_status_url(self):
             logging.error('no dtuvariant set')
         return url
 
+    def get_ahoy_base_url(self):
+        return f"http://{self.host}/api"
+
     def _refresh_data(self):
```

Comment:
docstring is missing, should we also create a baseurl function for opendtu?

Reply:
done. Incl. get_opendtu_base_url + get_template_base_url